src/main/kotlin/stringCalculator/Message.kt

```diff
@@ -0,0 +1,12 @@
+package stringCalculator
+
+class Message {
+    companion object {
```

Comment:
```suggestion
object Message {
```
클래스에 companion object만 있는 경우에는, object를 활용할 수 있어요.